core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-tables/contributor-admin-stats-table.component.ts

```diff
@@ -75,32 +73,14 @@ export class ContributorAdminStatsTable implements OnInit {
     filter: ContributorAdminDashboardFilter.createDefault(),
   };
 
-  columnsToDisplay = [
-    'chevron',
-    'contributorName',
-    'recentPerformance',
-    'overallAccuracy',
-    'submittedTranslationsCount',
-    'lastContributedInDays',
-    'role',
-  ];
-
-  dataSource:
-    | TranslationSubmitterStats[]
-    | TranslationReviewerStats[]
-    | QuestionSubmitterStats[]
-    | QuestionReviewerStats[] = [];
+  columnsToDisplay: string[] = [];
+
+  dataSource: ContributorStats[] = [];
 
   nextOffset: number = 0;
   more: boolean = true;
 
-  expandedElement:
-    | TranslationSubmitterStats[]
-    | TranslationReviewerStats[]
-    | QuestionSubmitterStats[]
-    | QuestionReviewerStats[]
-    | null
-    | [] = null;
+  expandedElement: ContributorStats[] | null | [] = null;
```

Comment:
Do you actually need the `[]` at the end -- wouldn't that be already accounted for in `ContributorStats[]`?

Reply:
Done